workflow/envs/picard_collect_wgs_metrics.yaml

```
@@ -0,0 +1,7 @@
+channels:
+  - conda-forge
+  - bioconda
+  - defaults
+dependencies:
+  - picard==2.22.1
```

should we maybe have same picard version as in the singularity 
```suggestion
  - picard==2.25.0
```